docs/notebooks/run_model_with_different_dataset_on_CRIB.ipynb

```diff
@@ -0,0 +1,317 @@
+    "#### On Snellius\n",
+    "- Site data:\n",
+    "    - path to data: `/projects/0/ttse0619/`,\n",
+    "    - config file template: https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/main/run_model_on_snellius/config_file_snellius.txt\n",
+    "- Global data:\n",
+    "    - path to data: `/projects/0/ttse0619/global_data`,\n",
+    "    - config file template: https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/main/run_model_on_snellius/config_file_snellius_global.txt\n",
+    "\n",
+    "Choose how do you want to run the model with MATLAB on CRIB:\n",
+    "\n",
+    "[**Using site data**](#1-using-site-data)\n",
+    "\n",
+    "[**Using global data**](#2-using-global-data)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Using site data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from PyStemmusScope import StemmusScope\n",
+    "from PyStemmusScope import save"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# user must provide the correct path\n",
+    "path_to_config_file = \"./config_file_crib.txt\"\n",
+    "path_to_model_src = \"/home/jovyan/private/Sarah/STEMMUS_SCOPE/src\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create an an instance of the model\n",
+    "model = StemmusScope(config_file=path_to_config_file, model_src_path=path_to_model_src, interpreter=\"Matlab\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "New config file /data/private/test_run/input/ZA-Kru_2024-01-29-1142/ZA-Kru_2024-01-29-1142_config.txt\n",
+      "Model input dir /data/private/test_run/input/ZA-Kru_2024-01-29-1142/\n",
+      "Model output dir /data/private/test_run/output/ZA-Kru_2024-01-29-1142/\n"
+     ]
+    }
+   ],
+   "source": [
+    "# setup the model\n",
+    "# here you can change the location and start & end time\n",
+    "config_path = model.setup(\n",
+    "    Location=\"ZA-Kru\",\n",
+    "    StartTime=\"2001-01-01T00:00\",\n",
+    "    EndTime=\"2001-01-02T01:30\",\n",
+    ")\n",
+    "\n",
+    "# new config file genertaed to run the model \n",
+    "print(f\"New config file {config_path}\")\n",
+    "\n",
+    "# see input and output paths generated by the model\n",
+    "print(f'Model input dir {model.config[\"InputPath\"]}')\n",
+    "print(f'Model output dir {model.config[\"OutputPath\"]}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Perform <echo $DISPLAY> in a terminal in the Remote Desktop (of Crib Platform), and check the output (should be similar to :1.0 ). Then execute the following line accordingly."
```

Comment:
What does this do? Why would someone want to do this

Reply:
This for CRIB users to be able to use MATLAB license manager while using jupyter interface. I will add more comments about it.